server/utils/tasks/seed-cache.ts

```diff
@@ -20,6 +20,13 @@ export async function seedCache() {
     totalApplicants: -1,
   }));
 
+  const activePostings = await db
+    .select({ totalApplicants: jobPostingsTable.totalApplicants })
+    .from(jobPostingsTable)
+    .where(eq(jobPostingsTable.isPublished, true));
+
+  const totalActivePostings = activePostings.length;
```

Comment:
you need to run `count` query to get count.

Reply:
So the goal is to get total applicants ? not active postings?

Reply:
you have to show posting count, not applicants count. Reason we need that from database is because we have future plans in cache management and paging side, so frontend won't be able to show length directly.